downstream/assemblies/platform/assembly-inventory-introduction.adoc

```diff
@@ -61,6 +61,8 @@ registry_password='<registry password>'
 
 The first part of the inventory file specifies the hosts or groups that Ansible can work with.
 
+For more information on `registry_username` and `registry_password`, see xref:proc-set-registry-username-password[Setting registry_username and registry_password].
```

Comment:
This assembly is in the planning guide: I'm getting an error when I build the planning guide.
(Installation guide builds OK)
Error:
`Unknown ID or title "proc-set-registry-username-password", used as an internal cross reference
/docs/repos/aap-docs/downstream/titles/aap-planning-guide/build/en-US/master.xml fails to validate`

Reply:
Had to guess at the link.